indico/web/client/styles/modules/_timetable.scss

```diff
@@ -101,6 +104,7 @@
     margin-top: -2px;
     height: 15px;
     width: 15px;
+    min-width: 15px;
```

Comment:
When the legend text is multi-line, the colored circle gets squished a bit. This forces it to stay circular